File: ALittle/Module/ALittleIDE/Script/UIEdit/IDEEditCore/IDESetting/ImagePlayS.lua

module("ALittleIDE", package.seeall)

local ___rawset = rawset
local ___pairs = pairs
local ___ipairs = ipairs
local ___coroutine = coroutine


assert(DisplayObjectS, " extends class:DisplayObjectS is nil")
ImagePlayS = ALittle.Class(DisplayObjectS, "ALittleIDE.ImagePlayS")

function ImagePlayS:Ctor(user_info, tab_child, tree_logic)
	___rawset(self, "_layer_name", "ide_setting_imageplay")
end

function ImagePlayS:LoadNatureBase()
	DisplayObjectS.LoadNatureBase(self)
	self:LoadValueData("interval")
	self:LoadShowTypeData("path_list")
end

function ImagePlayS:HandleIntervalFOCUSOUT(event)
	self:ValueNumZInputChange("interval", false)
end

function ImagePlayS:HandlePathListFOCUSOUT(event)
	self:TableDataSet("path_list", false)
end

File: ALittleIDE/Module/ALittleIDE/JSScript/IDE/UIEdit/Setting/ImagePlayS.js
{
if (typeof ALittleIDE === "undefined") window.ALittleIDE = {};


if (ALittleIDE.DisplayObjectS === undefined) throw new Error(" extends class:ALittleIDE.DisplayObjectS is undefined");
ALittleIDE.ImagePlayS = JavaScript.Class(ALittleIDE.DisplayObjectS, {
	Ctor : function(user_info, tab_child, tree_logic) {
		this._layer_name = "ide_setting_imageplay";
	},
	LoadNatureBase : function() {
		ALittleIDE.DisplayObjectS.LoadNatureBase.call(this);
		this.LoadValueData("interval");
		this.LoadShowTypeData("path_list");
	},
	HandleIntervalFOCUSOUT : function(event) {
		this.ValueNumZInputChange("interval", false);
	},
	HandlePathListFOCUSOUT : function(event) {
		this.TableDataSet("path_list", false);
	},
}, "ALittleIDE.ImagePlayS");

}